### PR TITLE
MixCalculator

### PR DESCRIPTION
**Pull request by PythonFZ**

### CODE REVIEW

File: ipsuite/calculators/mix.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    Calculator,
+    PropertyNotImplementedError,
+    all_changes,
+)
+
+from ipsuite import base
+from ipsuite.utils.ase_sim import freeze_copy_atoms
+
+
+def _update_is_exists(results, key, atoms_list, func, mean: bool):
```
Comment (M-R-Schaefer) [inline ipsuite/calculators/mix.py:16]: `update_if_exists`?

**Review by M-R-Schaefer — CHANGES_REQUESTED** [review]
possible minor spelling mistake

File: ipsuite/calculators/mix.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+        _update_is_exists(
+            self.results, "energy", sum_results, lambda x: x.get_potential_energy(), False
+        )
+        _update_is_exists(
+            self.results, "forces", sum_results, lambda x: x.get_forces(), False
+        )
+        _update_is_exists(
+            self.results, "stress", sum_results, lambda x: x.get_stress(), False
+        )
```
[the remark this answers is not in the document]
Comment (PythonFZ) [reply ipsuite/calculators/mix.py]: What do you think about the changes? If you have an idea, please show me how to make it look better.

[the remark this answers is not in the document]
Comment (PythonFZ) [reply ipsuite/calculators/mix.py]: had to rollback, ideas was https://github.com/zincware/IPSuite/pull/255/commits/bed696a9a4597abafc49736de487f7f94f027df7

**Pull request merged**